docs/workflow/building/coreclr/linux-instructions.md

```diff
@@ -114,27 +114,23 @@ If you are using Fedora, then you will need to install the following packages:
 Git Setup
 ---------
 
-This guide assumes that you've cloned the coreclr repository.
+This guide assumes that you've cloned the runtime repository.
 
 Set the maximum number of file-handles
 --------------------------------------
 
-To ensure that your system can allocate enough file-handles for the corefx build run `sysctl fs.file-max`. If it is less than 100000, add `fs.file-max = 100000` to `/etc/sysctl.conf`, and then run `sudo sysctl -p`.
-
-On Fedora:
-
-`$ sudo dnf install mono-devel`
+To ensure that your system can allocate enough file-handles for the libraries build run `sysctl fs.file-max`. If it is less than 100000, add `fs.file-max = 100000` to `/etc/sysctl.conf`, and then run `sudo sysctl -p`.
 
 Build the Runtime and Microsoft Core Library
```

Comment:
I don't think "Microsoft Core Library" is a commonly used term. If you want you could change such string(s) to System.Private.Corelib.